stable/datadog-controller/Chart.yaml

```diff
@@ -31,4 +31,4 @@ keywords:
 - datadog
 - controller
 sources:
-- https://github.com/deliveryhero/datadog-controller
```

Comment:
is changing `deliveryhero` for `max-rocket-internet` expected?

Reply:
Yes. I wrote it but in the end we never used it so it was just sitting under the DH org, unused. So I moved the repo to my own Github as I don't want some non-used, semi abandoned software left under DH org.

Reply:
🙌🏼